examples/pingpong/message.go

```diff
@@ -1,19 +1,26 @@
 package main
 
+import "github.com/renproject/phi"
+
 // Begin signals the pinger to start by sending a ping to the ponger.
-type Begin struct{}
+type Begin struct {
+	Responder chan phi.Message
+}
 
 // IsMessage implements the `phi.Message` interface.
 func (Begin) IsMessage() {}
 
 // Ping represents a ping.
-type Ping struct{}
+type Ping struct {
+	Responder chan phi.Message
+}
 
 // IsMessage implements the `phi.Message` interface.
 func (Ping) IsMessage() {}
 
 // Pong represents a pong.
-type Pong struct{}
+type Pong struct {
```

Comment:
See above comment about formatting
```suggestion
type Pong struct{}
```